taxi/forms.py

```diff
@@ -0,0 +1,30 @@
+from django import forms
+from .models import Car, Manufacturer
+from crispy_forms.helper import FormHelper
+from crispy_forms.layout import Submit
+
+
+class CarForm(forms.ModelForm):
+    class Meta:
+        model = Car
+        fields = ["model", "manufacturer", "drivers"]
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.helper = FormHelper()
+        self.helper.add_input(Submit("submit", "Save"))
+
+
+class ManufacturerForm(forms.ModelForm):
+    class Meta:
+        model = Manufacturer
+        fields = ["name", "country"]
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.helper = FormHelper()
+        self.helper.add_input(Submit("submit", "Save"))
+
+
+class DriverForm:
```

Comment:
The `DriverForm` class should extend `forms.ModelForm` to be consistent with the other forms and to function correctly as a Django form. Consider implementing it similarly to `CarForm` and `ManufacturerForm`.